### PR TITLE
Fix multi rooted issue and proper error handling

### PR DESCRIPTION
**Pull request by vives**

### CODE REVIEW

File: fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/util/ResultPayloadCreater.java [hunk 1]
```diff
@@ -41,10 +57,15 @@
 	 * @param element
 	 */
 	public void preparePayload(MessageContext messageContext, OMElement element) {
-
-		SOAPBody soapBody = messageContext.getEnvelope().getBody();
-		soapBody.addChild(element);
-
+        SOAPBody soapBody = messageContext.getEnvelope().getBody();
```
Comment (malakasilva) [inline fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/util/ResultPayloadCreater.java]: Invalid formatting please format properly

Comment (vives) [inline fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/util/ResultPayloadCreater.java]: Fixed

File: fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileDelete.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ public void connect(MessageContext messageContext) throws ConnectException {
 		try {
 			resultStatus = deleteFile(fileLocation, filename, filebeforepprocess);
 		} catch (FileSystemException e) {
-			generateResults(messageContext, resultStatus);
+			handleException("Error while Deleting a file/folder", e, messageContext);
```
Comment (malakasilva) [inline fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileDelete.java]: Found System.out.println("File deletion started..."); Maybe this is from a previous commit. Better remove it.

File: fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileExist.java [hunk 3]
```diff
@@ -63,7 +63,7 @@ public void connect(MessageContext messageContext) throws ConnectException {
 		try {
 			isFileExist = isFileExist(fileLocation, filename, content);
 		} catch (FileSystemException e) {
-			handleException(e.getMessage(), messageContext);
+			handleException(e.getMessage(),e, messageContext);
```
Comment (malakasilva) [inline fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileExist.java]: Why we are changing the style here. In earlier cases it's printing the error message.

**Pull request merged**